src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java

```diff
@@ -0,0 +1,64 @@
+/*  Copyright (C) 2012-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+package net.sf.jabref.logic.cleanup;
+
+import net.sf.jabref.model.entry.BibtexEntry;
+
+public class LatexCleanup {
```

Comment:
Please try to create a formatter without any dependencies to a BibtexEntry. The setting of the new value should be done in the `Cleanup` class whereas the formatter should also implement the `Formatter` interface. Have a look at other formatters already created, e.g., PageNumberFormatter.